### PR TITLE
Game update

### PR DESCRIPTION
**Pull request by svvoii**
This merge contains the following:
- Fixed paddle movement in single-player mode.
- Button to quit the game.
- Button to to trigger the game loop (ready btn)
- Refresh the GameBoard once the user quits the game or it ends. This allows you to start the new game session without refreshing the page manually. 
- Changed a bit the logic in `GameLogicAPI.js` and `GameBoard.js` (socket and initializeGame call are now handled in `GameBoard.js`
- Additional service for Nginx. It is not active - commented out in docker-compose (the project launches as you used to via HTTP).

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,17 +18,22 @@ services:
...
+
+  # nginx:
+  #   build:
+  #     context: ./nginx
+  #   container_name: nginx-proxy
+  #   ports:
+  #     - "80:80"
+  #     - "443:443"
+  #   depends_on:
+  #     - web
```
Comment (Drewski6) [inline docker-compose.yml:37]: I see some nginx configuration. Is this basically the config from your previous pull request #58 but with the portion in the docker-compose.yml file commented out so we can still develop on the school computers?
Just asking to verify I understand this portion
Thanks!

**Review by Drewski6 — APPROVED** [review]
Tested with 2 user in an online game. Looks good! Ready button is working.
Maybe this is for an upcomming pr but I noticed when I move the paddle on one screen, it did not move on the other, perhaps you're already working on this though. 
Otherwise looks great!

**Review by Roshanakk — APPROVED** [review]
Works for me !

File: docker-compose.yml (hunk 1 above)
Comment (svvoii) [reply docker-compose.yml:37]: That is correct. The Nginx part is there, just not active.

**Pull request merged**